.github/dependabot.yml

```
@@ -1,15 +1,15 @@
-# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
-
-version: 2
-updates:
-  - package-ecosystem: "gradle" # See documentation for possible values
-    directory: "/" # Location of package manifests
-    schedule:
-      interval: "weekly"
-    labels:
-      - "[Dependency Update]"
-    assignees:
+# To get started with Dependabot version updates, you'll need to specify which
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)